src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,90 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    public static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    public void setUp() {
+        user = new User();
+        user.setId(12345L);
+        user.setLogin("Sergey");
+        user.setPassword("123456");
+        user.setAge(20);
+    }
+
+    @Test
+    void registerUser_Ok() {
```

Comment:
about the naming of test methods:
![Screenshot 2022-07-04 at 08 49 00](https://user-images.githubusercontent.com/60258373/177251203-388375ab-2925-45c0-a4e8-1d8b30a2a769.png)